bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -115,6 +115,25 @@ spec:
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
 
+          check_db_health() {
+            db=$1
+            if [[ ! -f ${db} ]]; then
+                  echo "${db} does not exist, skipping health check"
+                  return
+            fi
+            echo "Checking $db health"
+            serverid=$(ovsdb-tool show-log ${db} | grep "server_id" | sed 's/server_id: \(.*\)/\1/')
+            match=$(ovsdb-tool show-log ${db} | grep "prev_servers:.*${serverid}(" || true)
```

Comment:
please quote `${db}`
can we avoid `.*`  and greediness backtracking and be more precise.  It should be just spaces.

`grep "prev_servers: *${serverid}(`
or
`grep "prev_servers:[[:blank:]]*${serverid}(`

Reply:
Done